frontend/test/card_info_util_test.dart

```diff
@@ -120,5 +154,113 @@ void main() {
           ..extensionNuernbergPassId = (NuernbergPassIdExtension()..passId = 99999999));
       expect(isCardNotYetValid(cardInfo), false);
     });
+
+    test('should return false if startDay is today and current time is 00:01 (still previous day in UTC)', () {
+      final berlinTimezone = getLocation('Europe/Berlin');
+
+      // Fake current time: 2025-06-06 00:01 in Berlin (2025-06-05 22:01 UTC)
+      withClock(Clock.fixed(TZDateTime(berlinTimezone, 2025, 6, 6, 0, 1)), () {
+        final startDay = DateTime.utc(2025, 6, 6);
+        final startDayEpoch = startDay.difference(DateTime.utc(1970, 1, 1)).inDays;
+
+        final cardInfo = CardInfo()
+          ..fullName = 'Max Mustermann'
+          ..expirationDay = 365 * 40
+          ..extensions = (CardExtensions()
+            ..extensionRegion = (RegionExtension()..regionId = 93)
+            ..extensionBirthday = (BirthdayExtension()..birthday = -365 * 10)
+            ..extensionNuernbergPassId = (NuernbergPassIdExtension()..passId = 99999999)
+            ..extensionStartDay = (StartDayExtension()..startDay = startDayEpoch));
+
+        expect(isCardNotYetValid(cardInfo), false);
+      });
+    });
+  });
+
+  group('isCardExpired', () {
+    test('should return false if expirationDay is tomorrow', () {
+      final cardInfo = CardInfo()
+        ..fullName = 'Max Mustermann'
+        ..expirationDay = getEpochDaysFromUtcOffset(1)
+        ..extensions = (CardExtensions()
+          ..extensionRegion = (RegionExtension()..regionId = 93)
+          ..extensionBirthday = (BirthdayExtension()..birthday = -365 * 10)
+          ..extensionNuernbergPassId = (NuernbergPassIdExtension()..passId = 99999999));
+      expect(isCardExpired(cardInfo), false);
+    });
+
+    test('should return true if expirationDay is today', () {
```

Comment:
❓ i guess the expired check in this pr didn't change, but i would expect if the expirationDay is today that my card is still valid and will be expired the day after expirationDay. Just a general idea

Reply:
I think you are right, I will adjust that 👌🏻